internal/cmd/connect/command_plugin_install.go

```diff
@@ -0,0 +1,445 @@
+package connect
+
+import (
+	"archive/zip"
+	"bytes"
+	"crypto/md5"
+	"crypto/sha1"
+	"encoding/json"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/spf13/cobra"
+
+	pcmd "github.com/confluentinc/cli/internal/pkg/cmd"
+	"github.com/confluentinc/cli/internal/pkg/cpstructs"
+	"github.com/confluentinc/cli/internal/pkg/errors"
+	"github.com/confluentinc/cli/internal/pkg/examples"
+	"github.com/confluentinc/cli/internal/pkg/form"
+	"github.com/confluentinc/cli/internal/pkg/output"
+	"github.com/confluentinc/cli/internal/pkg/utils"
+)
+
+const (
+	invalidDirectoryErrorMsg       = `plugin directory "%s" does not exist`
+	unexpectedInstallationErrorMsg = "unexpected installation type: %s"
+)
+
+func (c *pluginCommand) newInstallCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "install <plugin>",
+		Short: "Install a Connect plugin.",
+		Args:  cobra.ExactArgs(1),
+		RunE:  c.install,
+		Example: examples.BuildExampleString(
+			examples.Example{
+				Text: "Install the latest release of the Datagen connector onto your local Confluent Platform environment.",
+				Code: "confluent connect plugin install confluentinc/kafka-connect-datagen:latest",
+			},
+			examples.Example{
+				Text: "Install the latest release of the Datagen connector in a user specified directory and update one worker-config file.",
+				Code: "confluent connect plugin install confluentinc/kafka-connect-datagen:latest --plugin-directory $CONFLUENT_HOME/plugins --worker-configs $CONFLUENT_HOME/etc/kafka/connect-distributed.properties",
+			},
+		),
+		Annotations: map[string]string{pcmd.RunRequirement: pcmd.RequireOnPremLogin},
+	}
+
+	cmd.Flags().String("plugin-directory", "", "The plugin installation directory. If not specified, a default will be selected based on your Confluent Platform installation.")
+	cmd.Flags().StringSlice("worker-configs", []string{}, "A comma-separated list of paths to one or more Kafka Connect worker configuration files. Each worker file will be updated to load plugins from the plugin directory in addition to any preexisting directories.")
+	cmd.Flags().String("confluent-platform", "", "The path to a Confluent Platform archive installation. If not specified, this command will search for Confluent Platform installations in common locations.")
```

Comment:
> archive installation

Like a ZIP file? That doesn't sound right.

Reply:
```suggestion
	cmd.Flags().String("confluent-platform", "", "The path to a Confluent Platform archive installation. By default, this command will search for Confluent Platform installations in common locations.")
```

Reply:
Archive installation just means it was installed by the user from a zip or tar file, as opposed to using a package manager.